static/public/index.html

```diff
@@ -18,7 +18,7 @@ <h3>Test 2</h3>
     <!--Safei Playground-->
```

Comment:
The changes in this file are not mentioned in the description. Please clarify what these changes are for or should they be neglected.

Reply:
I have not made any changes to static folder or its content, was just checking the code I wrote and converting it into components copying some styles, probably a deleted comment or extra space, nothing that affected any kind of work on those files, please ignore it.